wolfssl/internal.h

```diff
@@ -692,7 +717,10 @@
         #if !defined(NO_RSA) && defined(HAVE_ECC)
             #define BUILD_TLS_ECDHE_RSA_WITH_CHACHA20_OLD_POLY1305_SHA256
         #endif
-        #if !defined(NO_DH) && !defined(NO_RSA)
+        #if !defined(NO_DH) && !defined(NO_RSA) && !defined(WSSL_HARDEN_TLS)
```

Comment:
Don't want to have WSSL_HARDEN_TLS everywhere so can we add a NO_TLS_DH define?

Reply:
Added a `NO_TLS_DH` define for the applicable checks.